pkg/kubelet/kubelet_node_status_test.go

```diff
@@ -785,7 +785,7 @@ func TestUpdateNodeStatusWithRuntimeStateError(t *testing.T) {
 		require.True(t, actions[1].Matches("patch", "nodes"))
 		require.Equal(t, actions[1].GetSubresource(), "status")
 
-		updatedNode, err := applyNodeStatusPatch(&existingNode, actions[1].(core.PatchActionImpl).GetPatch())
+		updatedNode, err := kubeClient.CoreV1().Nodes().Get(testKubeletHostname, metav1.GetOptions{})
```

Comment:
?

Reply:
The same as the other unit tests fixes. This test used to write its own Patch in `applyNodeStatusPatch`. Now it is not needed.